README.md

```diff
@@ -1,35 +1,94 @@
+## Overview
 
-You can use this image, if:
+To overcome these challenges, this image:
+- **Uses a Debian base** to ensure sorting behavior remains consistent with the standard PostgreSQL image.
+- **Includes the TimescaleDB extension** for time-series data support, without the issues introduced by Alpine-based builds.
 
-- If you want a PostgreSQL image with the Timeline extension that has the same sorting behavior as the standard PostgreSQL image.
+## Use Cases
 
-- If you are using the standard PostgreSQL image (not the `postgres:alpine` image) and want to keep your existing data volumes and just want to switch to a compatible image that just adds the Timescale extension.
+This image is ideal if:
+- You need a PostgreSQL image with the TimescaleDB extension that avoids the sorting issues associated with Alpine.
+- You are using the standard PostgreSQL image (not the `postgres:alpine` image) and want to seamlessly switch to an image with TimescaleDB, while maintaining compatibility with your existing data volumes.
 
-## How to use
+## Features
 
-Use the `Dockerfile` from this repository to build the image yourself or download from the GitHub container registry:
+- **Debian-Based Image**: Provides consistent sorting behavior and full compatibility with standard PostgreSQL images.
+- **Preinstalled TimescaleDB Extension**: Add time-series capabilities to your PostgreSQL database without additional setup.
+- **PostgreSQL Compatibility**: Designed to work seamlessly with existing PostgreSQL setups, ensuring an easy transition for users.
 
-```
+## How to Use
+
+### Pull the Image
+
+You can pull the prebuilt Docker image from the GitHub Container Registry:
+
+```bash
 docker pull ghcr.io/hoffmann-dsd/postgres-timescale:latest
 ```
 
-or more specific (recommended)
+For a specific version (recommended):
 
-```
+```bash
 docker pull ghcr.io/hoffmann-dsd/postgres-timescale:14.0.0
 ```
 
-Hint: The major version is aligned with the major version of the PostgreSQL version.
+### Build the Image Yourself
+
+If you prefer to build the image from this repository:
+
+```bash
+git clone https://github.com/hoffmann-dsd/postgres-timescale.git
+cd postgres-timescale
+docker build -t postgres-timescale:latest .
+```
+
+### Run the Container
+
+To run the container:
+
+```bash
+docker run -d \
+    -p 5432:5432 \
+    --name postgres-timescale \
+    -e POSTGRES_PASSWORD=yourpassword \
+    ghcr.io/hoffmann-dsd/postgres-timescale:latest
+```
+
+### Volumes and Data Persistence
+
+For persistent storage, mount your data volume:
+
+```bash
+docker run -d \
+    -p 5432:5432 \
+    -v /path/to/your/data:/var/lib/postgresql/data \
+    --name postgres-timescale \
+    -e POSTGRES_PASSWORD=yourpassword \
+    ghcr.io/hoffmann-dsd/postgres-timescale:latest
+
+```
+
+## License
+
+This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for more details.
+
+## Known Issues
```

Comment:
"Known Issues" should list issues with this solution and not the issues that this project solves. I would remove this section.